schedule/containers/extra_tests_textmode_containers.yaml

```diff
@@ -22,10 +22,12 @@ conditional_schedule:
         DISTRI:
             'opensuse':
                 - containers/docker_image
+                - containers/container_diff
```

Comment:
@Vogtinator as you mentioned it makes no sense to schedule this outside of x86_64 architecture. Can you suggest me how to add this as a second condition here?

Reply:
No idea. It's one of the reasons why I dislike yaml schedules.

Reply:
As `container-diff` is actually already for platforms as well, we don't need this any more.